src/transformers/generation/logits_process.py

```diff
@@ -323,6 +320,88 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
         return scores
 
 
+class EpsilonLogitsWarper(LogitsWarper):
+    r"""
+    [`LogitsWarper`] that performs epsilon, i.e. restricting to tokens with absolute prob > prob_cut_off. Takes the
```

Comment:
```suggestion
    [`LogitsWarper`] that performs epsilon-sampling, i.e. restricting to tokens with absolute prob > prob_cut_off. Takes the
```

(you might need to run `make fixup` locally and commit again, after incorporating this change)